Tasks/NodeToolV0/nodetool.ts

```diff
@@ -17,9 +17,11 @@ async function run() {
         let versionSpecInput = taskLib.getInput('versionSpec', versionSource == 'spec');
         let versionFilePathInput = taskLib.getInput('versionFilePath', versionSource == 'fromFile');
         let nodejsMirror = taskLib.getInput('nodejsMirror', false);
+        const retryCountOnDownloadFails = taskLib.getInput('retryCountOnDownloadFails', false) || "5";
+        const delayBetweenRetries = taskLib.getInput('delayBetweenRetries', false) || "1000";
         let versionSpec = getNodeVersion(versionSource, versionSpecInput, versionFilePathInput);
         let checkLatest: boolean = taskLib.getBoolInput('checkLatest', false);
-        await getNode(versionSpec, checkLatest, nodejsMirror.replace(/\/$/, ''));
+        await getNode(versionSpec, checkLatest, nodejsMirror.replace(/\/$/, ''), parseInt(retryCountOnDownloadFails), parseInt(delayBetweenRetries));
```

Comment:
Why not [parseFloat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseFloat)?

Reply:
retryCountOnDownloadFails - is the count of retries which logically should be integer
delayBetweenRetries - is in milliseconds, so couldn't be used as a float as well